Tests/SwiftFusionTests/Geometry/Pose2Tests.swift

```diff
@@ -1,6 +1,24 @@
-import SwiftFusion
+import TensorFlow
 import XCTest
 
+import SwiftFusion
+
+func randomPose2() -> Pose2 {
```

Comment:
Make a method Random in Pose2? Arbitrary tangentvector with given covariance matrix and then move :-)

Reply:
done